app/controllers/chargeback_controller.rb

```diff
@@ -247,6 +248,9 @@ def cb_rate_edit
                   detail.group = r[:group]
                   detail.per_unit = r[:per_unit]
                   detail.metric = r[:metric]
+                  # Copy the measure id of the rate_detail linkig with the rate_detail_measure
+                  id_measure = ChargebackRateDetailMeasure.find_by(:name => r[:measure]).id
```

Comment:
If the record is not found, then `find_by` will return a nil, and the `.id` call will be an undefined method `id` for NilClass.  Are we guranteed to have the measure in question?